### PR TITLE
added pagination to index page

### PR DESCRIPTION
**Pull request by jsc215**

### CODE REVIEW

**Comment by kyjoya** [comment]
I'm going to merge this, but consider refactoring to create a reusable "pagination" component.

File: react/src/containers/SuperheroesIndexContainer.js [hunk 1]
```diff
@@ -42,11 +55,37 @@ class SuperheroesIndexContainer extends Component {
...
     });
+
+    let pageNumbers = [];
+    for (let i = 1; i <= Math.ceil(this.state.superheroes.length / this.state.heroesPerPage); i++) {
+      if (this.state.currentPage === i) {
+        pageNumbers.push(`[${i}]`);
+      } else {
+        pageNumbers.push(i);
+      }
+
```
Comment (jdmacmurtrie) [inline react/src/containers/SuperheroesIndexContainer.js]: Remove this blank line

File: spec/controllers/api/v1/reviews_controller_spec.rb [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+  #         }
+  #       }
+  #
+  #     current_reviews_count = Review.count
+  #     post :create, params: params
+  #     expect(Review.count).to eq(current_reviews_count + 1)
+  #     expect(response).to have_http_status :ok
+  #   end
+  # end
+
```
Comment (jdmacmurtrie) [inline spec/controllers/api/v1/reviews_controller_spec.rb]: Is this being used or not?

File: spec/controllers/api/v1/superheroes_controller_spec.rb [hunk 3]
```diff
@@ -54,4 +54,47 @@
...
+  #     session[:user_id] = kjoya.id
+  #     post :create, params: params
+  #     # superhero: {name: 'daredevil', backstory: 'blind and stuff', superpower: 'ninja skills', image_url: 'image.png', user: kjoya}
+  #     # expect{ post(:create, params: params)}.to change{ Superhero.count }.by 1
+  #     # binding.pry
+  #     expect(Superhero.count).to eq(current_superhero_count + 1)
+  #     expect(response).to have_http_status :ok
+  #   end
+  # end
+
```
Comment (jdmacmurtrie) [inline spec/controllers/api/v1/superheroes_controller_spec.rb]: What about this one?

File: spec/controllers/reviews_controller_spec.rb [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+  }
+
+  let!(:review) {
+    Review.create(
+      rating: 3,
+      comment: "Hallo",
+      superhero_id: hulk.id,
+      user_id: kjoya.id
+    )
+  }
```
Comment (jdmacmurtrie) [inline spec/controllers/reviews_controller_spec.rb]: Y'all realize factoryBot would make this a lot easier right?

File: spec/controllers/reviews_controller_spec.rb [hunk 5]
```diff
@@ -0,0 +1,48 @@
...
+  #     expect(response).to render_template('index')
+  #   end
+  # end
+  # describe 'GET show' do
+  #   it 'should render show template' do
+  #     get :show, params: { id: review.id }
+  #     expect(response).to render_template('show')
+  #   end
+  # end
+end
```
Comment (jdmacmurtrie) [inline spec/controllers/reviews_controller_spec.rb]: Either delete commented code or use it, please

**Pull request merged**